app/connectors/LastTaxCalculationConnector.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import javax.inject.{Inject, Singleton}
+
+import models._
+import play.api.Logger
+import play.api.http.Status
+import play.api.http.Status._
+import uk.gov.hmrc.play.config.ServicesConfig
+import uk.gov.hmrc.play.http.{HeaderCarrier, _}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+@Singleton
+class LastTaxCalculationConnector @Inject()(val http: HttpGet) extends ServicesConfig with RawResponseReads {
+
+  lazy val protectedMicroserviceUrl: String = baseUrl("income-tax-view-change")
+  lazy val getEstimatedTaxLiabilityUrl: String => String = nino => s"$protectedMicroserviceUrl/income-tax-view-change/estimated-tax-liability/$nino"
+
+  def getLastEstimatedTax(nino: String)(implicit headerCarrier: HeaderCarrier): Future[LastTaxCalculationResponseModel] = {
+
+    val url = getEstimatedTaxLiabilityUrl(nino)
+    Logger.debug(s"[LastEstimatedTaxCalculationConnector][getLastEstimatedTax] - GET $url")
+
+    http.GET[HttpResponse](url) flatMap {
+      response =>
+        response.status match {
```

Comment:
For now we have agree to render ISE - separate story to handle this scenario gracefully